app/scripts/directives/pageMessage.dir.js

```diff
@@ -0,0 +1,27 @@
+'use strict';
+/*
+  To use the pageMessage function, call the PageMessage Service from the state's controller.
+*/
+
+angular.module('yourStlCourts').directive('pageMessage', function($rootScope){
+  return {
+    restrict: 'E',
+    templateUrl: 'views/pageMessage.html',
+    controller: 'PageMessageCtrl as ctrl',
+    link: function ($scope, element, attrs) {
```

Comment:
I'm not sure I follow you.  Ordinarily I could use an ng-class, but this case seemed different.  This directive is used on the index.html page and that page does not have a controller.  And as far as I know I can't check on an angular service (i.e. PageMessageSvc) from html using ng-class.  So, I have to use a controller to do the check.  And since the index page does not have a controller, I use the directive's controller to read the service.